### ISSUE TITLE
[BUG] Another black screen issue

### ISSUE THREAD

**Issue by itsFoxee** [PojavLauncherTeam/PojavLauncher#2949]
### Describe the bug

Fresh install, wanted to try this amazing piece of software out. I wanted to join a 1.18.1 server, downloaded the recommended by you Java 17 repository for arm64. Set it as default, went to launch 1.18.1, and then, black screen. Thought, maybe it's just loading a bit longer, and waited. After about a minute, not just the Minecraft window, the whole app crashes. Like, exits to the last app I've been on. Complete hard stop. Though I wanted to see if other versions have issues, I've launched 1.14.4, everything works well. I've also tried all renderers, to no avail. Is 1.18.1 just too new of a version and it doesn't have support yet? All help is appreciated

```
---- Minecraft Crash Report ----
// Quite honestly, I wouldn't worry myself about that.

Time: 3/21/22, 8:45 AM
Description: Initializing game

java.lang.IllegalStateException: could not preload shader position
	at epe.a(SourceFile:423)
	at epe.a(SourceFile:407)
	at dxo.<init>(SourceFile:617)
	at net.minecraft.client.main.Main.main(SourceFile:199)
Caused by: yk: Invalid shaders/core/position.json: Couldn't compile vertex program (minecraft:shaders/core/position.vsh, position) : 0:7: S0054: Overloading built-in function 'max' not allowed
	at yk.a(SourceFile:48)
	at epw.<init>(SourceFile:200)
	at epe.a(SourceFile:419)
	... 3 more
Caused by: java.io.IOException: Couldn't compile vertex program (minecraft:shaders/core/position.vsh, position) : 0:7: S0054: Overloading built-in function 'max' not allowed
	at dry.b(SourceFile:72)
	at dry.a(SourceFile:54)
	at epw.a(SourceFile:241)
	at epw.<init>(SourceFile:182)
	... 4 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at epe.a(SourceFile:423)
	at epe.a(SourceFile:407)
	at dxo.<init>(SourceFile:617)

-- Initialization --
Details:
	Modules: 
Stacktrace:
	at net.minecraft.client.main.Main.main(SourceFile:199)

-- System Details --
Details:
	Minecraft Version: 1.18.1
	Minecraft Version ID: 1.18.1
	Operating System: Linux (aarch64) version Android-8.1.0
	Java Version: 17-internal, N/A
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 705300296 bytes (672 MiB) / 2172649472 bytes (2072 MiB) up to 2172649472 bytes (2072 MiB)
	CPUs: 4
	Processor Vendor: 0x41
	Processor Name: AArch64 Processor rev 4 (aarch64)
	Identifier: 0x41 Family 8 Model 0xd03 Stepping r0x0p4
	Microarchitecture: unknown
	Frequency (GHz): -0.00
	Number of physical packages: 1
	Number of physical CPUs: 1
	Number of logical CPUs: 8
	Graphics card #0 name: unknown
	Graphics card #0 vendor: unknown
	Graphics card #0 VRAM (MB): 0.00
	Graphics card #0 deviceId: unknown
	Graphics card #0 versionInfo: unknown
	Virtual memory max (MB): 3945.69
	Virtual memory used (MB): 4568.99
	Swap memory total (MB): 1024.00
	Swap memory used (MB): 349.32
	JVM Flags: 2 total; -Xms2072M -Xmx2072M
	Launched Version: 1.18.1
	Backend library: LWJGL version 3.2.3 SNAPSHOT
	Backend API: GL4ES wrapper GL version 2.1 gl4es wrapper 1.1.5, ptitSeb
	Window size: <not initialized>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: <disabled>
	Using VBOs: Yes
	Is Modded: Probably not. Client jar signature and brand is untouched
	Type: Client (map_client.txt)
	CPU: 8x AArch64 Processor rev 4 (aarch64)
```

### The log file and images/videos

[latestlog.txt](https://github.com/PojavLauncherTeam/PojavLauncher/files/8314132/latestlog.txt)

### Steps To Reproduce

```markdown
1. Start PojavLauncher.
2. Login using Microsoft.
3. Select 1.18.1.
4. Launch 1.18.1.
5. Dark screen, wait ~1 min.
6. App exits.
```

### Expected Behavior

Expected to launch 1.18.1 in the app's environment 

### Platform

```markdown
- Device model: CUBOT POWER
- CPU architecture: arm64
- Android version: 8.1.0
- PojavLauncher version: crocus-v3_openjdk
```

### Anything else?

-

**Comment by itsFoxee** [comment]
Update: Tried once again. Now the app crashes and displays "Game exited."

**Comment by 1493028** [comment]
> Update: Tried once again. Now the app crashes and displays "Game exited."

Try to reboot your phone

**Comment by itsFoxee** [comment]
I've rebooted my phone, unfortunately the issue still persists. I'm starting to think it's because my phone's computing power is lacking. Other versions like 1.8.9 work just fine

**Comment by ENDERMANYK** [comment]
Follow the steps here: 
https://docs.google.com/document/d/18QJUfh-WANGbNI3cZyY3FK55MzqnH2KIAaZzxF1NrXE/edit?usp=sharing

**Comment by GaucovyAero** [comment]
Could not preload shader position indicates that you haven't yet patched your 1.18.1 jar file. Just follow the guide ENDERMANYK sent and you will get it working :)

**Comment by itsFoxee** [comment]
This is true! All I needed was the patcher. Works fine. Thank you!
